native/src/hooks/useEmbeddedOffers.ts

```diff
@@ -0,0 +1,56 @@
+import { useCallback } from 'react'
+
+import {
+  CategoryModel,
+  LoadSprungbrettJobReturn,
+  SPRUNGBRETT_OFFER,
+  loadSprungbrettJobs,
+  useLoadAsync,
+} from 'api-client'
+
+import { determineApiUrl } from '../utils/helpers'
+
+type EmbeddedOffers = {
+  sprungbrett?: Awaited<LoadSprungbrettJobReturn>
+}
+
+type EmbeddedOffersParams = {
+  category?: CategoryModel | null
+  cityCode: string
+  languageCode: string
+}
+
+export type EmbeddedOffersReturn = {
+  embeddedOffers: EmbeddedOffers | null
+  error: Error | null
+  loading: boolean
+  refresh: () => void
+}
```

Comment:
Optional: Is there a reason to use custom return value with `embeddedOffers` instead of `data` property? I think this would simplify things.
```suggestion
export type EmbeddedOffersReturn = ReturnType<EmbeddedOffers>
```